daemon/algod/api/algod.oas3.yml

```diff
@@ -1375,6 +1375,10 @@
             "pattern": "^(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?$",
             "type": "string"
           },
+          "round": {
+            "description": "The round for which this information is relevant",
+            "type": "integer"
+          },
```

Comment:
@jasonpaulos would it be better to add the round in BoxResponse instead?  BoxResponse uses a schema reference though, and I seem to remember something about not being able to add to a schema reference.

Reply:
No, this makes sense here. The schema ref that BoxResponse uses means it's really just an alias to Box. See our generated type here: https://github.com/algorand/go-algorand/blob/ebcd07b302d0f26d85ae538fbf1c2d620e7bed2e/daemon/algod/api/server/v2/generated/model/types.go#L858